lib/xpi/utils.js

```diff
@@ -6,6 +6,8 @@
 var join = require("path").join;
 var when = require("when");
 var fs = require("fs-promise");
+var Zip = require("jszip");
```

Comment:
@kumar303 I choose ```jszip``` mostly because it is what ```zip-dir``` uses internally.
I'm not sure about the issues on #410 but it can definitely used to directly manipulate zip files.